src/linear_algebra/determinant.lean

```diff
@@ -312,8 +355,12 @@ end
 /-- Specialization of `linear_equiv.is_unit_det` -/
 lemma linear_equiv.is_unit_det' {A : Type*} [comm_ring A] [is_domain A] [module A M]
   (f : M ≃ₗ[A] M) : is_unit (linear_map.det (f : M →ₗ[A] M)) :=
-by haveI := classical.dec_eq M; exact
-(f : M →ₗ[A] M).det_cases (λ s b, f.is_unit_det _ _) is_unit_one
+is_unit_of_mul_eq_one _ _ f.det_mul_det_symm
+
+/-- The determinant of `f.symm` is the inverse of that of `f` when `f` is a linear equiv. -/
+lemma linear_equiv.det_coe_symm {𝕜 : Type*} [field 𝕜] [module 𝕜 M]
+  (f : M ≃ₗ[𝕜] M) : (f.symm : M →ₗ[𝕜] M).det = (f : M →ₗ[𝕜] M).det ⁻¹ :=
+by field_simp [is_unit.ne_zero f.is_unit_det']
```

Comment:
Does this need the `field` assumption? I think you can prove this via `linear_equiv.det_symm` and `linear_equiv.coe_det` with only `comm_ring R`.

Reply:
Oh nevermind, of course it does because otherwise there's no `⁻¹` to speak about!